lib/plugins/Label_5Z_Slash.ts

```
@@ -48,12 +49,15 @@ export class Label_5Z_Slash extends DecoderPlugin {
 
     const lines = message.text.split('\r\n');
     if (lines[0] === '/TXT') {
-      // not UA, but starts with `/`
-      ResultFormatter.text(decodeResult, lines.slice(1).join('\r\n'));
-      decodeResult.decoded = true;
-      decodeResult.decoder.decodeLevel = 'full';
-      return decodeResult;
-    }
+    // not UA, but starts with `/`
```

ESLint

[error] Insert `··`

(prettier/prettier)